packages/core/database/lib/index.d.ts

```diff
@@ -160,6 +160,15 @@ export interface Database {
   entityManager: EntityManager;
 
   query<T extends keyof AllTypes>(uid: T): QueryFromContentType<T>;
+  transaction(
+    cb?: (params: {
+      trx: Knex.Transaction;
+      rollback: () => Promise<void>;
+      commit: () => Promise<void>;
+    }) => Promise<void>
+  ):
+    | Promise<unknown>
```

Comment:
I'm assuming this is the equivalent of `ReturnType<typeof trx.run>`? What does the unknown means here? (basically, what does a trx.run can return?)

Reply:
it returns what the callback returns, which could be anything

Reply:
Thanks :slightly_smiling_face: